test/integration/targets/azure_rm_lock/aliases

```diff
@@ -0,0 +1,5 @@
+cloud/azure
+shippable/azure/group3
+destructive
+disabled
```

Comment:
Use `unsupported` instead of `disabled` since the test will never run in CI.

Reply:
So, `unsupported` means that this module cannot be tested in the CI. `disabled` means the CI tests are temporarily disabled. Do I misunderstand?

Reply:
@mattclay Could you please provide some details?  If PR is added with has_issue label, which items should be checked for cause? Thank you very much!

Reply:
@MyronFanQiu You are correct about the `unsupported` and `disabled` aliases.

@Fred-sun The `has_issue` label does not mean anything is wrong with the PR. It simply means the PR has an issue associated with it.

Reply:
@mattclay Thanks for your explanation. Can you help to trigger the CI verify again? the error is that a  test item is unstable. Thank you very much!